expression/mvcc.go

```diff
@@ -0,0 +1,62 @@
+package expression
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"sync/atomic"
+
+	"github.com/go-sql-driver/mysql"
+)
+
+var (
+	portGenerator       uint32 = 4000
+	statusPortGenerator uint32 = 10080
```

Comment:
`StatusPort` can be config. So some server may not be `10080`

Reply:
How can I get the `StatusPort` from config?